src/main/java/com/weblyzard/sparql/StreamingResultSet.java

```diff
@@ -162,4 +162,9 @@ public int getRowNumber() {
     public void close() throws IOException {
         in.close();
     }
+    
+    private static String escape(String in) {
+        String result = in.replaceAll("\"\"", "\\\\\\\"");
+        return result;
```

Comment:
can you please add a pointer to the documentation suggesting to escape nodes using this schema.